pkg/gen/gen.go

```diff
@@ -68,7 +68,7 @@ func execute(ctx context.Context, file input.Input) error {
 func isRegenerable(path string) bool {
```

Comment:
I thought the `zz_generated.` prefix determines if a file can be re-generated. Why do we need to identify by file name?

Reply:
Some file names are conventional. Like `Makefile` or `dependabot.yml`. We still want to regenerate them.

Reply:
Makes sense. Good morning Tim. 😅